react/features/app-settings/components/AppSettings.native.js

```diff
@@ -56,110 +55,88 @@ class AppSettings extends AbstractAppSettings {
     render() {
         const { _profile, t } = this.props;
 
-        // FIXME: presentationStyle is added to workaround orientation issue on
-        // iOS
-
         return (
             <Modal
                 animationType = 'slide'
                 onRequestClose = { this._onRequestClose }
-                presentationStyle = 'overFullScreen'
+                presentationStyle = 'fullScreen'
                 supportedOrientations = { [
                     'landscape',
                     'portrait'
                 ] }
                 visible = { this.props._visible }>
-                <View
-                    style = { [
-                        styles.headerContainer,
-                        this._getSafetyPadding()
-                    ] } >
-                    <BackButton
-                        onPress = { this._onRequestClose }
-                        style = { styles.settingsBackButton } />
-                    <Text style = { [ styles.text, styles.headerTitle ] } >
-                        { t('profileModal.header') }
-                    </Text>
+                <View style = { PlatformElements.page }>
+                    <Header>
+                        <BackButton
+                            onPress = { this._onRequestClose } />
+                        <Text
+                            style = { [
+                                styles.text,
+                                PlatformElements.headerText
+                            ] } >
+                            { t('profileModal.header') }
```

Comment:
can we please rename all these xxxModal to something else? having "Modal" in the name doesn't really add anything and may become incorrect in the future.